### ISSUE TITLE
Enabling ScriptBlockLogging should not be a part of JEAEndpoint setup

### ISSUE THREAD

**Issue by jnury** [ChrisLGardner/JeaDsc#6]
When using JeaEndpoint to create a new endpoint, ScriptBlockLogging is activated.

First, it's not correctly implemented as the registry key property is not tested. So it may become 'not compliant' but will not be detected by DSC. Also, even if you remove the endpoint, registry value will not be restored.

ScriptBlockLogging is an interesting feature. But I think we have to let users choosing whether they want to enable it or not. In my recent experience, enabling scriptblock logging disclosed some private information as poorly designed scripts handled sensitive data with clear text parameters :-/

I suggest removing this part of the Set function and let users activate the logging with a Registry resource.

**Comment by ChrisLGardner** [comment]
I agree we shouldn't be doing that in this resource, it should be mentioned in the Readme that its not done and needs to be implemented by using the Registry resource or similar.
________________________________
